gallery/src/components/NxPagination/NxPaginationPage.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxPaginationExample from './NxPaginationExample';
+import NxPaginationEmptyExample from './NxPaginationEmptyExample';
+import NxPaginationSinglePageExample from './NxPaginationSinglePageExample';
+
+const nxPaginationCode = require('!!raw-loader!./NxPaginationExample').default,
+    nxPaginationEmptyCode = require('!!raw-loader!./NxPaginationEmptyExample').default,
+    nxPaginationSinglePageCode = require('!!raw-loader!./NxPaginationSinglePageExample').default;
+
+const NxPaginationPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        Pagination controls are used when there is a great deal of content that needs to be sorted into separate
+        "pages" rather than scrolled. The controls offer multiple methods of navigating the paged content, forward and
+        backward controls, specific page controls, and an ellipsis control.
+        This component renders pagination buttons contained within an <code className="nx-code">.nx-btn-bar</code>.
+        It is intended to be used within an <code className="nx-code">.nx-footer</code> element.
+        There are essentially five different "types" of buttons which can appear within the pagination control:
+      </p>
+      <ul className="nx-list">
+        <li className="nx-list__item">
+          <span className="nx-list__text">Current page button</span>
+          <span className="nx-list__subtext">
+            A button for the current page will always be visible and is distinguished via a dark blue background.
+            Clicking this button has no effect - the component's <code className="nx-code">onChange</code> handler
+            does not get triggered.
+          </span>
+        </li>
+        <li className="nx-list__item">
+          <span className="nx-list__text">Next/previous page buttons</span>
+          <span className="nx-list__subtext">
+            Whenever there are pages either above or below the current page, arrow buttons are rendered as
+            applicable to navigate the pages sequentially.
+          </span>
+        </li>
+        <li className="nx-list__item">
+          <span className="nx-list__text">Nearby page buttons</span>
+          <span className="nx-list__subtext">
+            Buttons for up to 4 additional pages that precede or follow the current page will be present as
+            applicable. The lowest "nearby page" to get a rendered button always has a page index that is one
```

Comment:
```suggestion
            Buttons for up to 4 additional pages that precede or follow the current page will be present. The lowest "nearby page" to get a rendered button always has a page index that is one
```

Reply:
> \+ Buttons for up to 4 additional pages that precede or follow the current page will be present. The lowest "nearby page" to get a rendered button always has a page index that is one

Again, "as applicable" was important here.  If you're close to the end of list of pages you might not get a full 4 nearby pages.